src/wallet/scriptpubkeyman.cpp

```diff
@@ -1385,21 +1385,21 @@ void LegacyScriptPubKeyMan::ReturnDestination(int64_t nIndex, bool fInternal, co
     WalletLogPrintf("keypool return %d\n", nIndex);
 }
 
-bool LegacyScriptPubKeyMan::GetKeyFromPool(CPubKey& result, const OutputType type, bool internal)
+bool LegacyScriptPubKeyMan::GetKeyFromPool(CPubKey& result, const OutputType type)
 {
     assert(type != OutputType::BECH32M);
-    if (!CanGetAddresses(internal)) {
+    if (!CanGetAddresses(/*internal=*/ false)) {
         return false;
     }
 
     CKeyPool keypool;
     {
         LOCK(cs_KeyStore);
         int64_t nIndex;
-        if (!ReserveKeyFromKeyPool(nIndex, keypool, internal) && !m_storage.IsWalletFlagSet(WALLET_FLAG_DISABLE_PRIVATE_KEYS)) {
+        if (!ReserveKeyFromKeyPool(nIndex, keypool, /*fRequestedInternal=*/ false) && !m_storage.IsWalletFlagSet(WALLET_FLAG_DISABLE_PRIVATE_KEYS)) {
```

Comment:
I'm guessing that just dropping the space will fix the CI issue?

```cpp
/*fRequestedInternal=*/false
```

Reply:
CI is passing now after rename `internal` to `fRequestedInternal`. CI checks below appear all green to me.

Reply:
Oh you're right, I missed the last push.

In the majority of cases we don't use a space after `=*/`, but that's not worth touching the PR for imo.